NOTES.create-cmd_states.db3

```diff
@@ -1,4 +1,9 @@
-To make the fresh sqlite3 version of relevant command states Sybase tables:
+To make a fresh sqlite3 version of command states Sybase tables:
 
-$ ./make_new_tables.py --tstart=1999:001 --cmds --cmd-states  # takes a few minutes
+$ ./make_new_tables.py --tstart=1999:001 --cmds --cmd-states # takes a few minutes
+
+To make a complete sqlite3 version to use for on-the-side sqlite3 crontasks, do:
+
+# ./make_new_tables.py --tstart=1999:001 --cmds --cmd-states --tl_processing # takes a few minutes
```

Comment:
The comment is an accident.  

 The two versions apparently come from my misunderstanding of your requirements.  I thought you might have some other driver for wanting to copy out the cmd_states and timelines to a new table (a portable reference database?).  Relevant was not really defined in the original text.  If you want to copy the database so it it can be used as an independent database that works with the three scripts to update cmd_states  you need to copy over the other tables (the tl_* ones).    If you have no reason to want to just copy out the timelines